infra/terraform/terraform_backend/terraform.tfstate

```diff
@@ -1,103 +1,96 @@
 {
   "version": 4,
```

Comment:
Not sure why this was checked in  but we could move this our `cft-infra-test-tfstate` backend.

Reply:
Per discussion removing terraform_backend and retaining the legacy `cloud-foundation-cicd-tfstate` and `cloud-foundation-cicd-seed-projects-tfstate` buckets.